hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlan.java

```diff
@@ -177,30 +178,31 @@ public void initialize(NodeEngine nodeEngine,
             QueuedPipe<Object>[] snapshotQueues = new QueuedPipe[vertex.localParallelism()];
             Arrays.setAll(snapshotQueues, i -> new OneToOneConcurrentArrayQueue<>(SNAPSHOT_QUEUE_SIZE));
             ConcurrentConveyor<Object> ssConveyor = ConcurrentConveyor.concurrentConveyor(null, snapshotQueues);
+            ILogger storeSnapShotLogger = new PrefixedLogger(
+                    nodeEngine.getLogger(StoreSnapshotTasklet.class), sanitizeLoggerNamePart(vertex.name()));
```

Comment:
We no longer need to sanitize the vertex name, now it's a part of the message, not of the logger name. 

Same on line 251, I can't comment there because it's not changed in this PR. We can also remove the `sanitizeLoggerNamePart()` method, we're no longer using any user input text in logger names anywhere.